### ISSUE TITLE
"Play/Pause" controls are not provided for the "carousel slider" in "Visual studio code" page:A11y_VisualStudioCodeServices_Getting started_overview_pause,stop,hide

### ISSUE THREAD

**Issue by PrasannaPolisetty123** [microsoft/vscode-docs#7411]
## GitHub Tags:
#A11yTCS; #A11ySev2; #A11yMAS; #DesktopWeb; #ChromiumEdge; #WCAG2.2.2; #pausestophide;#BM_VisualStudioCodeServices_Web_July2024; #Visual Studio Code Services; #Benchmark; #Win11; #FTP; #Desktopweb;#Ailimited;

## Environment Details: 
Application Name:  Visual Studio Code Services
URL: https://code.visualstudio.com/docs/?dv=win64user&dark-plus-v2
Microsoft Edge version 126.0.2592.68 (Official build) (64-bit)

## Repro steps:

1. Launch the application URL https://code.visualstudio.com/download?dark-plus-v2 and press ENTER.
2. TAB to "carousel slider" under the "visual studio code in action "and observe that "play/pause" control is not provided for the slider.
## Actual:
"carousel slider" is continuously changing and play/pause control is not provided for the "carousel slider"
## Expected:
play/pause control should be provided for the "carousel slider" so users can view the template and access the information present in the template.
## User Impact:
Low vision users will get impacted if the "play/pause" controls are not provided for the "carousel slider", hence they will miss the information and not able to read the content clearly if the sliders are moving without play/pause control.
## Attachment:
![bullet](https://github.com/microsoft/vscode-docs/assets/151031203/d7a371db-64c2-4a45-87ed-33db927b34aa)

**Comment by 12-shweta** [comment]
@daviddossett could you please provide the environment to verify this bug.

#Rev:shwarh;

**Comment by PrasannaPolisetty123** [comment]
@daviddossett 
The expected fix 
https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/

**Comment by 12-shweta** [comment]
#A11yPreview;

**Comment by SwatiJo** [comment]
This issue still reproes, there is no mechanism provided to pause or stop carousel  
![Recording-20240627_123114](https://github.com/microsoft/vscode-docs/assets/167469595/00af2bbb-40a3-444f-a0a4-15f36146133d)

**Comment by daviddossett** [comment]
The image above looks like the wrong URL. You're missing the `?dark-plus-v2` query param on the URL. This is what it should look like. I removed the carousel entirely.

![Image](https://github.com/user-attachments/assets/2bd22cf9-cf37-4598-9e83-5802af615b43)

**Comment by SwatiJo** [comment]
@PrasannaPolisetty123  Issue has been fixed. Carousel is no more present on the page. 
![Recording-20240627_132631](https://github.com/microsoft/vscode-docs/assets/167469595/36c72b67-b22e-4e87-8473-f1ec6c308228)

**Comment by PrasannaPolisetty123** [comment]
closing the bug as carousel is removed
![image](https://github.com/microsoft/vscode-docs/assets/151031203/f23b3564-280d-4a5e-858a-d4f00a2fb233)
